src/wasm/multipart.rs

```diff
@@ -9,6 +9,7 @@ use web_sys::FormData;
 use super::Body;
 
 /// An async multipart/form-data request.
+#[derive(Clone)]
```

Comment:
Likewise, I'd probably keep these _public_ promises away, in case we wanted to change implementations later and couldn't make them `Clone`.